sass/chosen.scss

```diff
@@ -8,6 +8,9 @@ $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
   vertical-align: middle;
   font-size: 13px;
   user-select: none;
+  &:focus {
+    outline: none;
```

Comment:
@adunkman Just to double check: this just removes some extra extraneous focus on some unexpected element, right?  Chosen currently styles properly on focus like you'd expect (good for accessibility!), so I'm hoping this is just that!

Reply:
Correct! This removes a focus rectangle around the Chosen DIV itself, which only occurs when clicking on search results (after the search input has blurred but before the search result is selected). It’s quite jarring, since it’s an unexpected element to receive focus in this case.